frontend/amundsen_application/static/js/ducks/feature/reducer.ts

```diff
@@ -99,6 +171,24 @@ export default function reducer(
         statusCode: action.payload.statusCode,
         feature: action.payload.feature,
       };
+    case GetFeatureDescription.FAILURE:
+    case GetFeatureDescription.SUCCESS:
+      return {
+        ...state,
+        feature: {
+          ...state.feature,
+          description: action.payload.description,
```

Comment:
should we add a loading state on this state?

Reply:
This is a non-blocking call for the user. When enter the edit mode, it checks to see if the description has been updated by another user since we last fetched this description.